code/CodeTester.cpp

```diff
@@ -456,7 +456,39 @@ meta_unicast_locator.port = 7412;
 participant_attr.rtps.builtin.metatrafficUnicastLocatorList.push_back(meta_unicast_locator);
 //!--
 
-//CONF_DISCOVERY_SERVER
+//CONF_DISCOVERY_SERVER_MAIN_SCENARIO
+
+//CONF_DS_MAIN_SCENARIO_SERVER
+Locator_t server_locator;
+IPLocator::setIPv4(server_locator, "192.168.10.57");
+server_locator.port = 56542;
+
+RemoteServerAttributes server_attr;
+server_attr.ReadguidPrefix("72.61.73.70.66.61.72.6d.74.65.73.74");
+server_attr.metatrafficUnicastLocatorList.push_back(server_locator);
+
+participant_attr.rtps.builtin.discovery_config.discoveryProtocol = DiscoveryProtocol_t::SERVER;
+participant_attr.rtps.builtin.discovery_config.m_DiscoveryServers.push_back(server_attr);
```

Comment:
This is not needed nor correct. You'd do something like this to connect one server to another
```suggestion
```